tests/test_helpers.py

```diff
@@ -504,6 +505,15 @@ def test_send_from_directory(self):
             assert rv.data.strip() == b'Hello Subdomain'
             rv.close()
 
+    def test_send_from_directory_bad_request(self):
+        app = flask.Flask(__name__)
+        app.testing = True
+        app.root_path = os.path.join(os.path.dirname(__file__),
+                                     'test_apps', 'subdomaintestmodule')
+        with app.test_request_context():
+            with pytest.raises(BadRequest):
+                rv = flask.send_from_directory('static', 'bad\x00')
+                rv.close()
```

Comment:
This shouldn't be necessary - the previous line already raises an exception so you never get here